<launch>
  <!-- Arguments -->
  <arg name="move_forward_only" default="false"/>


  <!-- move_base -->
  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <param name="base_local_planner" value="dwa_local_planner/DWAPlannerROS" />
    <rosparam file="$(find my_nav)/params/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find my_nav)/params/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find my_nav)/params/map_nav_params/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find my_nav)/params/map_nav_params/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find my_nav)/params/move_base_params.yaml" command="load" />
    <rosparam file="$(find my_nav)/params/dwa_local_planner_params_dbot.yaml" command="load" />
    <param name="DWAPlannerROS/min_vel_x" value="0.0" if="$(arg move_forward_only)" />
  </node>

</launch>
